index.html

```diff
@@ -11771,9 +11771,10 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 			<div class="state-description">
 				<p><a>Indicates</a> whether the <a>element</a> is exposed to an accessibility API. See related <sref>aria-disabled</sref>.</p>
 				<p>User agents determine an element's [=element/hidden=] status based on whether it is rendered, and the rendering is usually controlled by CSS. For example, an element whose <code>display</code> property is set to <code>none</code> is not rendered. An element is considered [=element/hidden=] if it, or any of its ancestors are not rendered or have their <code>aria-hidden</code> attribute value set to <code>true</code>.</p>
-				<p>Authors MAY, with caution, use aria-hidden to hide visibly rendered content from assistive technologies <em>only</em> if the act of hiding this content is intended to improve the experience for users of assistive technologies by removing redundant or extraneous content. Authors using aria-hidden to hide visible content from screen readers MUST ensure that identical or equivalent meaning and functionality is exposed to assistive technologies.</p>
+				<p>Authors MUST NOT use <code>aria-hidden</code> to hide root elements of a document. For instance, the <code>html</code> or <code>body</code> elements in an HTML document. Otherwise, authors MAY, with caution, use aria-hidden to hide visibly rendered content from assistive technologies <em>only</em> if the act of hiding this content is intended to improve the experience for users of assistive technologies by removing redundant or extraneous content. Authors using aria-hidden to hide visible content from screen readers MUST ensure that identical or equivalent meaning and functionality is exposed to assistive technologies.</p>
```

Comment:
Root element is the right term for `<html>` but maybe not `<body>`. I don't recall what the term is but it maybe listed somewhere in an XML or HTML spec.

Reply:
The [HTML Spec for the body](https://html.spec.whatwg.org/multipage/sections.html#the-body-element) element includes:
> The [body](https://html.spec.whatwg.org/multipage/sections.html#the-body-element) element [represents](https://html.spec.whatwg.org/multipage/dom.html#represents) the contents of the document.

So this may work:
```suggestion
				<p>Authors MUST NOT use <code>aria-hidden</code> to hide the root element or the host language element that <a href="https://html.spec.whatwg.org/multipage/dom.html#represents">represents</a> the contents of a document. For instance, the <code>html</code> or <code>body</code> elements in an HTML document. Authors MAY, with caution, use <code>aria-hidden</code> to hide visibly rendered content from assistive technologies <em>only</em> if the act of hiding this content is intended to improve the experience for users of assistive technologies by removing redundant or extraneous content. Authors using <code>aria-hidden</code> to hide visible content from screen readers MUST ensure that identical or equivalent meaning and functionality is exposed to assistive technologies.</p>
```